src/KafkaFlow/Consumers/OffsetCommitter.cs

```diff
@@ -13,11 +13,14 @@ internal class OffsetCommitter : IOffsetCommitter
         private readonly ILogHandler logHandler;
 
         private readonly Timer commitTimer;
+        private readonly IReadOnlyList<Timer> statisticsTimers;
```

Comment:
I would create just one handler to this statistic, many Timers can lead to performance degradation